types/types.go

```diff
@@ -198,6 +198,7 @@ type Info struct {
 	Images             int
 	Driver             string
 	DriverStatus       [][2]string
+	CustomStatus       [][2]string
```

Comment:
@calavera we want the output to be displayed in the same order it was added. You're right that this is effectively a map, but using a slice of slices gives more control to the tool using `CustomStatus` wrt how it will be displayed.